gtk4/src/param_spec_expression.rs

```diff
@@ -84,15 +86,26 @@ impl FromGlibPtrFull<*mut gobject_ffi::GParamSpec> for ParamSpecExpression {
 impl ParamSpecExpression {
     #[allow(clippy::new_ret_no_self)]
     #[doc(alias = "gtk_param_spec_expression")]
-    pub fn new(name: &str, nick: &str, blurb: &str, flags: glib::ParamFlags) -> ParamSpec {
+    pub fn new(
+        name: impl IntoGStr,
+        nick: impl IntoGStr,
+        blurb: impl IntoGStr,
```

Comment:
These are optional or not? `IntoOptionalGStr`

Reply:
Yes they are optional